history/characters/10000_orc.txt

```diff
@@ -572,15 +572,15 @@
 		effect = {
 			set_relation_friend = character:68001 #Vol'jin
 
-			set_variable = { name = wc_endurance_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_3_physical_trait_value }
-			set_variable = { name = wc_strength_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_3_physical_trait_value }
-			set_variable = { name = wc_elemental_fire_magic_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_magic_trait_value }
-			set_variable = { name = wc_elemental_water_magic_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_magic_trait_value }
-			set_variable = { name = wc_elemental_air_magic_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_magic_trait_value }
-			set_variable = { name = wc_elemental_earth_magic_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_magic_trait_value }
+			set_variable = { name = wc_endurance_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_2_physical_trait_value }
```

Comment:
I don't get what is point from the changelog.

Reply:
Changed Thrall's and Remulos' class levels according to requests in #731

Added that in changelog